CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,129 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
```

Comment:
While I understand the idea I would definitely refrain from using _race_ here. The term itself is inappropriate for humans and everything that this could refer to is - in my opinion - already covered (i.e. ethnicity, nationality, personal appearance).
The 2.1 version of the contributor covenant has two interesting additions to the list: _caste_ and _color_. While I believe _caste_ is even worse than _race_, _color_ might express the original intent best.

[Wikipedia](https://en.wikipedia.org/wiki/Race_(human_categorization)) says:
> The concept of race is foundational to [racism](https://en.wikipedia.org/wiki/Racism), the belief that humans can be divided based on the superiority of one race over another.

Reply:
Totally agreed here. I've made the changes.